deps.edn

```diff
@@ -1 +1 @@
-{:deps {honeysql {:mvn/version "0.9.5" :exclusions [org.clojure/clojurescript]}}}
+{:deps {honeysql {:mvn/version "1.0.444" :exclusions [org.clojure/clojurescript]}}}
```

Comment:
Should we do this upgrade in isolation?

Reply:
I'd bumped it up to work with the latest clause priorities.